prow/plugins/override/override.go

```diff
@@ -304,6 +304,21 @@ type descriptionAndState struct {
 	state       string
 }
 
+// Parses /override foo something
+func parseOverrideInput(in string) []string {
+	// This func does the following:
+	// return ["tide"] when tide is passed
+	// return ["foo","bar"] when foo bar is passed
+	// return ["style / Lint"] "style / Lint" is passed
+	if regexp.MustCompile(`^\"(.+)\"$`).MatchString(in) {
```

Comment:
The current supported patterns are:
- `/override "<GH Action A>"` but not `/override "<GH Action A>" "<GH Action B>"`
- `/override prowjob-c prowjob-d GH-Action-With-no-space-in-name`

imho this is not consistent behavior and would cause confusion to users. I might be wrong but it doesn't sound too hard to derive contexts names from something like `"<GH Action A>" prowjob-c "<GH Action B>"`, is there anything that I missed here?

Reply:
This `GH-Action-With-no-space-in-name` doesn't really exist especially with [matrixes](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs). `/override prowjob-c prowjob-d GH-Action-With-no-space-in-name` is not a thing.

Rightnow, I'm blocked so I'm happy with just restoring my feature without reverting the PR. Also, I'm bit bad at regex :(

Reply:
sorry to hear that you're blocked. this function doesn't really need regex imo, there are a few methods mentioned in https://stackoverflow.com/questions/47489745/splitting-a-string-at-space-except-inside-quotation-marks, among them `strings.FieldsFunc` is more readable to me